pyannote/core/annotation.py

```diff
@@ -472,6 +473,85 @@ def crop(self, support: Support, mode: CropMode = 'intersection'):
             else:
                 raise NotImplementedError("unsupported mode: '%s'" % mode)
 
+    def truncate(self, removed: Support, mode: CropMode = 'intersection') \
```

Comment:
I'd rather call it `extrude`.

Reply:
Yes! I actually hesited on using `extrude` instead of `truncate`, but wondered if it wasn't too related to 3D stuff. I'll change that.